modules/platforms/dotnet/Apache.Ignite/SslStreamFactory.cs

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Apache.Ignite;
+
+using System.ComponentModel;
+using System.IO;
+using System.Net.Security;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+using Internal.Common;
+
+/// <summary>
+/// Standard SSL stream factory. Can be used with or without client-side certificates.
+/// </summary>
+public sealed class SslStreamFactory : ISslStreamFactory
+{
+    /// <summary>
+    /// Default SSL protocols.
+    /// </summary>
+    public const SslProtocols DefaultSslProtocols = SslProtocols.None;
+
+    /// <summary>
+    /// Gets or sets the certificate file (pfx) path.
+    /// <para />
+    /// Java certificates can be converted with <c>keytool</c>:
+    /// <c>keytool -importkeystore -srckeystore thekeystore.jks -srcstoretype JKS
+    /// -destkeystore thekeystore.pfx -deststoretype PKCS12</c>.
```

Comment:
Not sure this is a right place for this kind of info.

Reply:
Good point, this was carried over from Ignite 2.x, fixed.